e2e-tests/gef/cypress/fixtures/mocks/mock-facilities.js

```diff
@@ -1,7 +1,24 @@
 const CONSTANTS = require('../constants');
 const dateConstants = require('../../../../e2e-fixtures/dateConstants');
 
+const facilityEndDateEnabled = Number(Cypress.env('GEF_DEAL_VERSION')) >= 1;
+
+/**
+ * Gets facility end date properties if enabled on default deal version
+ * @returns {Pick<import('@ukef/dtfs2-common').Facility,'isUsingFacilityEndDate' | 'facilityEndDate'> | {} } mock facility end date properties if enabled, or empty object if not
```

Comment:
I'm slightly uncertain about how this type works - what does `Pick` mean?

Reply:
`Pick<T, 'propertyName'>` will generate a sub-type (is that a word?) of `T` that has the property `T`

So in this case I'm returning the type `{ isUsingFacilityEndDate: boolean; facilityEndDate: Date }`